Week 3

FranklinMar opened the pull request:
new file:   ft_printf/printf_test/ft_printf.c
	new file:   libft/Makefile
	new file:   libft/ft_abs.c
	new file:   libft/ft_abs.o
	new file:   libft/ft_bzero.c
	new file:   libft/ft_div.c
	new file:   libft/ft_isalpha.c
	new file:   libft/ft_isascii.c
	new file:   libft/ft_isdigit.c
	new file:   libft/ft_memccpy.c
	new file:   libft/ft_memchr.c
	new file:   libft/ft_memcmp.c
	new file:   libft/ft_memcpy.c
	new file:   libft/ft_memmove.c
	new file:   libft/ft_memset.c
	new file:   libft/ft_strchr.c
	new file:   libft/ft_strdup.c
	new file:   libft/ft_striter.c
	new file:   libft/ft_strjoin.c
	new file:   libft/ft_strmap.c
	new file:   libft/ft_strncmp.c
	new file:   libft/ft_strnstr.c
	new file:   libft/ft_strrchr.c
	new file:   libft/ft_strsplit.c
	new file:   libft/ft_strstr.c
	new file:   libft/ft_strsub.c
	new file:   libft/ft_strtrim.c
	new file:   libft/ft_tolower.c
	new file:   libft/ft_toupper.c
	new file:   libft/libft.a
	new file:   libft/libft.h
	new file:   tester/.vscode/settings.json
	new file:   tester/Makefile
	new file:   tester/README.md
	new file:   tester/tests/ft_abs_test.cpp
	new file:   tester/tests/ft_bzero_test.cpp
	new file:   tester/tests/ft_div_test.cpp
	new file:   tester/tests/ft_isalpha_test.cpp
	new file:   tester/tests/ft_isascii_test.cpp
	new file:   tester/tests/ft_isdigit_test.cpp
	new file:   tester/tests/ft_memccpy_test.cpp
	new file:   tester/tests/ft_memchr_test.cpp
	new file:   tester/tests/ft_memcmp_test.cpp
	new file:   tester/tests/ft_memcpy_test.cpp
	new file:   tester/tests/ft_memmove_test.cpp
	new file:   tester/tests/ft_memset_test.cpp
	new file:   tester/tests/ft_strchr_test.cpp
	new file:   tester/tests/ft_strdup_test.cpp
	new file:   tester/tests/ft_striter_test.cpp
	new file:   tester/tests/ft_strjoin_test.cpp
	new file:   tester/tests/ft_strmap_test.cpp
	new file:   tester/tests/ft_strncmp_test.cpp
	new file:   tester/tests/ft_strnstr_test.cpp
	new file:   tester/tests/ft_strrchr_test.cpp
	new file:   tester/tests/ft_strsplit_test.cpp
	new file:   tester/tests/ft_strstr_test.cpp
	new file:   tester/tests/ft_strsub_test.cpp
	new file:   tester/tests/ft_strtrim_test.cpp
	new file:   tester/tests/ft_tolower_test.cpp
	new file:   tester/tests/ft_toupper_test.cpp
	new file:   tester/utils/check.cpp
	new file:   tester/utils/check.hpp
	new file:   tester/utils/color.cpp
	new file:   tester/utils/color.hpp
	new file:   tester/utils/leaks.cpp
	new file:   tester/utils/leaks.hpp
	new file:   tester/utils/sigsegv.cpp
	new file:   tester/utils/sigsegv.hpp

FranklinMar committed:
Changes to be committed:
	new file:   ft_printf/printf_test/ft_printf.c
	new file:   libft/Makefile
	new file:   libft/ft_abs.c
	new file:   libft/ft_abs.o
	new file:   libft/ft_bzero.c
	new file:   libft/ft_div.c
	new file:   libft/ft_isalpha.c
	new file:   libft/ft_isascii.c
	new file:   libft/ft_isdigit.c
	new file:   libft/ft_memccpy.c
	new file:   libft/ft_memchr.c
	new file:   libft/ft_memcmp.c
	new file:   libft/ft_memcpy.c
	new file:   libft/ft_memmove.c
	new file:   libft/ft_memset.c
	new file:   libft/ft_strchr.c
	new file:   libft/ft_strdup.c
	new file:   libft/ft_striter.c
	new file:   libft/ft_strjoin.c
	new file:   libft/ft_strmap.c
	new file:   libft/ft_strncmp.c
	new file:   libft/ft_strnstr.c
	new file:   libft/ft_strrchr.c
	new file:   libft/ft_strsplit.c
	new file:   libft/ft_strstr.c
	new file:   libft/ft_strsub.c
	new file:   libft/ft_strtrim.c
	new file:   libft/ft_tolower.c
	new file:   libft/ft_toupper.c
	new file:   libft/libft.a
	new file:   libft/libft.h
	new file:   tester/.vscode/settings.json
	new file:   tester/Makefile
	new file:   tester/README.md
	new file:   tester/tests/ft_abs_test.cpp
	new file:   tester/tests/ft_bzero_test.cpp
	new file:   tester/tests/ft_div_test.cpp
	new file:   tester/tests/ft_isalpha_test.cpp
	new file:   tester/tests/ft_isascii_test.cpp
	new file:   tester/tests/ft_isdigit_test.cpp
	new file:   tester/tests/ft_memccpy_test.cpp
	new file:   tester/tests/ft_memchr_test.cpp
	new file:   tester/tests/ft_memcmp_test.cpp
	new file:   tester/tests/ft_memcpy_test.cpp
	new file:   tester/tests/ft_memmove_test.cpp
	new file:   tester/tests/ft_memset_test.cpp
	new file:   tester/tests/ft_strchr_test.cpp
	new file:   tester/tests/ft_strdup_test.cpp
	new file:   tester/tests/ft_striter_test.cpp
	new file:   tester/tests/ft_strjoin_test.cpp
	new file:   tester/tests/ft_strmap_test.cpp
	new file:   tester/tests/ft_strncmp_test.cpp
	new file:   tester/tests/ft_strnstr_test.cpp
	new file:   tester/tests/ft_strrchr_test.cpp
	new file:   tester/tests/ft_strsplit_test.cpp
	new file:   tester/tests/ft_strstr_test.cpp
	new file:   tester/tests/ft_strsub_test.cpp
	new file:   tester/tests/ft_strtrim_test.cpp
	new file:   tester/tests/ft_tolower_test.cpp
	new file:   tester/tests/ft_toupper_test.cpp
	new file:   tester/utils/check.cpp
	new file:   tester/utils/check.hpp
	new file:   tester/utils/color.cpp
	new file:   tester/utils/color.hpp
	new file:   tester/utils/leaks.cpp
	new file:   tester/utils/leaks.hpp
	new file:   tester/utils/sigsegv.cpp
	new file:   tester/utils/sigsegv.hpp

josadcha commented:
First of all the commit name is wrong. Commit name should describe the content of this commit, for example: [Week3] Cunix2 module exercises.

josadcha commented:
I'm not able to run `make` inside `libft` directory, even `.o` files are not generated. Please fix Makefile, otherwise I won't be able to test it.
![image](https://user-images.githubusercontent.com/39953699/129535499-7ffb5b21-53bc-45aa-bbc7-e7e9922a471e.png)

FranklinMar committed:
modified:   cunix2/libft/Makefile
	deleted:    cunix2/libft/ft_abs.o
	modified:   cunix2/libft/ft_memmove.c
	modified:   cunix2/libft/ft_strchr.c
	modified:   cunix2/libft/ft_strjoin.c
	modified:   cunix2/libft/ft_strncmp.c
	modified:   cunix2/libft/ft_strnstr.c
	modified:   cunix2/libft/ft_strrchr.c
	modified:   cunix2/libft/ft_strstr.c
	modified:   cunix2/libft/ft_strtrim.c
	modified:   cunix2/libft/libft.a
